tests/vmcheck/test-misc-2.sh

```diff
@@ -56,6 +56,9 @@ fi
 vm_cmd journalctl --after-cursor "'$from_cursor'" -u rpm-ostreed -o json | jq -r '.AGENT//""' > agent.txt
```

Comment:
Looking at this, I think it was meant to be

```suggestion
vm_cmd journalctl --after-cursor "'$cursor'" -u rpm-ostreed -o json | jq -r '.AGENT//""' > agent.txt
```

While you're here, want to fix that up?